app/jobs/regular/export_user_archive.rb

```diff
@@ -153,6 +155,35 @@ def badges_export
       end
     end
 
+    def bookmarks_export
+      return enum_for(:bookmarks_export) unless block_given?
+
+      Bookmark
+        .where(user_id: @current_user.id)
+        .joins(:post)
+        .order(:id)
+        .each do |bkmk|
+        link = ''
+        if guardian.can_see_post?(bkmk.post)
+          link = bkmk.post.full_url
+        end
```

Comment:
This could be a one-liner?

```suggestion
        link = bkmk.post.full_url if guardian.can_see_post?(bkmk.post)
```

Reply:
@riking feel free to commit that style change directly